src/sections/app.style.js

```diff
@@ -35,8 +36,11 @@ body {
   line-height: 28px;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  font-family: 'Open Sans', sans-serif;
-  color: ${(props) => props.theme.textColor};
+
+  color: ${(props) => props.theme.text};
+  font-family: Tahoma, Helvetica, Arial, Roboto, sans-serif;
```

Comment:
This should be removed

Reply:
I still don't see this as removed

Reply:
I think this should not be removed as it select default text for dark and white theme..

Reply:
This is incorrect font-family and we don't use it anywhere. So, it should be removed.

Reply:
Oh sorry my bad , I thought you were talking about that textColor